python/ql/src/semmle/python/Stmts.qll

```diff
@@ -90,8 +90,10 @@ class AugAssign extends AugAssign_ {
     /* syntax: Expr += Expr */
     override Expr getASubExpression() { result = this.getOperation() }
 
+    /** Gets the target of this augmented assignment statement. */
     Expr getTarget() { result = this.getOperation().(BinaryExpr).getLeft() }
 
+    /** Gets the value of this augmented assignment statement. */
```

Comment:
Would that be `1` in `x += 1`? That is the RHS but not the final value of `x` so potentially a bit confusing. Thus I think the comment should spell it out.

Reply:
Agreed. I've updated the comment.